extensions/python/src/main/java/com/hazelcast/jet/python/PythonServiceContext.java

```diff
@@ -261,4 +265,8 @@ private static void createParamsScript(@Nonnull Path paramsFile, String... names
             }
         }
     }
+
+    public BiFunctionEx<String, Integer, ? extends ManagedChannelBuilder<?>> channelFn() {
```

Comment:
Missing `NonNull`

Reply:
It's a private API, we don't bother. Many other methods in the class also don't have it.